lookout/style/typos/generation.py

```diff
@@ -388,8 +388,7 @@ def _load_tree(self, tree: dict) -> None:
         self.checker._deletes = deletes
         self.checker._words = {w: self.checker._words[i] for i, w in enumerate(words)}
         vectors = self.wv["vectors"]
-        wv = FastTextKeyedVectors(vectors.shape[1], self.wv["min_n"], self.wv["max_n"],
-                                  self.wv["bucket"], True)
+        wv = FastTextKeyedVectors(vectors.shape[1], self.wv["min_n"], self.wv["max_n"])
```

Comment:
We shouldn't change this. Most likely this caused problems for you because you have outdated version of gensim. The requirements were updated recently, you should use the version stated there.